db/migrate/20210516011732_create_company_users.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCompanyUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :company_users do |t|
+      t.references :company, null: false, foreign_key: {on_delete: :cascade}
```

Comment:
Rubocop didn't get it?
```suggestion
      t.references :company, null: false, foreign_key: { on_delete: :cascade }
```

Reply:
Nop.

Reply:
Then please adjust it to keep our pattern :D